$OpenBSD: patch-mk_modules_mk,v 1.1 2021/06/15 06:54:44 landry Exp $

otherwise echo tries to link twice its objfile and that results in conflicting exports

Index: mk/modules.mk
--- mk/modules.mk.orig
+++ mk/modules.mk
@@ -277,7 +277,6 @@ MODULES   += contact
 MODULES   += ctrl_tcp
 MODULES   += debug_cmd
 MODULES   += ebuacip
-MODULES   += echo
 MODULES   += fakevideo
 MODULES   += httpd
 MODULES   += ice
